ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -590,6 +590,19 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: e2e-osd-ccs-nightly-aws
```

Comment:
Suggest to ask Productivity Team to review. cc @liangxia

Reply:
Communicate with @yasun1 on slack, they only have 1 or 2 jobs there. We have agreement that it's fine to put it there at the moment. They will migrate to a dedicated job yaml in the future if the job number increase significantly.
Thank you for letting me know @jianlinliu